DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/CourseDelegates/Index.cshtml

```diff
@@ -33,19 +33,11 @@
              asp-route-sortDirection="@Model.CourseDetails?.SortDirection"

```

Comment:
Can't comment it against the line, but I think we should probably update the id & the corresponding javascript that uses it now that the button is not Export Current